app/models/version.rb

```diff
@@ -257,6 +265,7 @@ def payload
       'prerelease'       => prerelease,
       'licenses'         => licenses,
       'requirements'     => requirements,
+      'runtime_dependencies_count' => runtime_dependencies_count,
```

Comment:
What about to lazily counter cache those values (on version create or counter access)?